tracing-subscriber/tests/reload.rs

```diff
@@ -79,3 +90,51 @@ fn reload_handle() {
         assert_eq!(FILTER2_CALLS.load(Ordering::SeqCst), 1);
     })
 }
+
+#[test]
+fn reload_filter() {
```

Comment:
i think it's actually just this test that requires the `registry` feature, so let's just put the feature flag here instead?
```suggestion
#[cfg(feature = "registry")]
fn reload_filter() {
    struct NopLayer;
    impl<S: Subscriber> tracing_subscriber::Layer<S> for NopLayer {
        fn register_callsite(&self, _m: &Metadata<'_>) -> Interest {
            Interest::sometimes()
        }

        fn enabled(&self, _m: &Metadata<'_>, _: layer::Context<'_, S>) -> bool {
            true
        }
    }

```

Reply:
...wait, my bad --- there is no `reload` feature. so it seems that these tests actually weren't being run on `master`. lmao.